--- zbar-0.23/configure.ac.orig	2019-07-24 16:28:37.917511357 +0200
+++ zbar-0.23/configure.ac	2019-07-24 16:33:05.382729041 +0200
@@ -693,7 +693,7 @@
   [],
   [with_java="check"])
 
-JAVAC=${JAVAC/ecj/ecj -1.5}
+JAVAC=$(echo "$JAVAC" | sed -e 's/ecj/ecj -1.5/')
 
 # Javah was obsoleted on Java 8 and removed on Java 11. So, we need to
 # look strictly at the $JAVA_HOME in order to avoid mixing different versions
