--- zbar-0.20/configure.ac.orig	2017-05-14 09:01:27.438180076 +0200
+++ zbar-0.20/configure.ac	2017-05-14 09:04:18.028178129 +0200
@@ -91,7 +91,7 @@
 
 AC_ARG_VAR([JAVAC], [location of Java language compiler])
 AC_PATH_PROGS([JAVAC], [javac jikes ecj gcj], [:], [$JAVA_PATH])
-JAVAC=${JAVAC/ecj/ecj -1.5}
+JAVAC=$(echo "$JAVAC" | sed -e 's/ecj/ecj -1.5/')
 AS_IF([test "x$JAVAC" = "x:"], [have_java="no"])
 
 AC_ARG_VAR([JAVAH], [location of Java header generator])
