--- zbar-0.23.1/pygtk/zbarpygtk.override.orig	1970-01-01 01:00:00.000000000 +0100
+++ zbar-0.23.1/pygtk/zbarpygtk.override	2020-09-12 22:11:40.240520389 +0200
@@ -0,0 +1,20 @@
+%%
+headers
+#include <Python.h>
+#include <pygobject.h>
+#include <zbar/zbargtk.h>
+%%
+modulename zbarpygtk
+%%
+import gtk.Widget as PyGtkWidget_Type
+import gtk.gdk.Pixbuf as PyGdkPixbuf_Type
+%%
+ignore-type
+  ZBarGtkError
+%%
+ignore-glob
+  *_get_type
+%%
+ignore
+  zbar_gtk_image_from_pixbuf  # until base library wrappers are in place
+%%
