diff -urNp -x '*.orig' zbar-0.23.90.org/pygtk/zbarpygtk.override zbar-0.23.90/pygtk/zbarpygtk.override
--- zbar-0.23.90.org/pygtk/zbarpygtk.override	1970-01-01 01:00:00.000000000 +0100
+++ zbar-0.23.90/pygtk/zbarpygtk.override	2021-03-02 23:13:40.469676611 +0100
@@ -0,0 +1,20 @@
+%%
+headers
+#include <Python.h>
+#include <pygobject.h>
+#include <zbar/zbargtk.h>
+%%
+modulename zbarpygtk
+%%
+import gtk.Widget as PyGtkWidget_Type
+import gtk.gdk.Pixbuf as PyGdkPixbuf_Type
+%%
+ignore-type
+  ZBarGtkError
+%%
+ignore-glob
+  *_get_type
+%%
+ignore
+  zbar_gtk_image_from_pixbuf  # until base library wrappers are in place
+%%
