diff -urNp -x '*.orig' zbar-0.23.90.org/configure.ac zbar-0.23.90/configure.ac
--- zbar-0.23.90.org/configure.ac	2021-02-13 13:56:31.000000000 +0100
+++ zbar-0.23.90/configure.ac	2021-03-02 23:13:39.453013275 +0100
@@ -728,7 +728,7 @@ AC_ARG_WITH([java],
   [],
   [with_java="check"])
 
-JAVAC=${JAVAC/ecj/ecj -1.5}
+JAVAC=$(echo "$JAVAC" | sed -e 's/ecj/ecj -1.5/')
 
 dnl Javah was obsoleted on Java 8 and removed on Java 11. So, we need to
 dnl look strictly at the $JAVA_HOME in order to avoid mixing different versions
